src/main.rs

```diff
@@ -225,10 +234,42 @@ fn settings() -> io::Result<Settings> {
         None => DEFAULT_DELETION_CHUNK_SIZE,
     };
 
+    // validates the input argument parsed into --older-than, and converts the &str to a Duration
```

Comment:
The [`parse_duration`](https://docs.rs/parse_duration/latest/parse_duration/) crate looks pretty good. Maybe we can use that?

Reply:
I was not aware of this crate, thank you for the heads up. It has now been implemented.